pandas/core/indexes/accessors.py

```diff
@@ -126,6 +126,26 @@ class DatetimeProperties(Properties):
     """
 
     def to_pydatetime(self):
+        """
+        Return DatetimeIndex as an object ndarray.
```

Comment:
I would mention it is an array of "native Python datetime objects" ?

Reply:
Can you change this to "Return Series" (as it is the accessor on a Series) ?

Reply:
done, Kindly review